### PR TITLE
Merge README from Amp-Swift repo

### PR DESCRIPTION
**Pull request by olha-si**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,19 +91,19 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
...
-amp.observe("Signup", ["lang": "en", "timeOfDay": "evening"])
+amp.observe("CheckoutAmount", ["amount": amount])
 ```
 
 >Objective-C
 ```ObjectiveC
-[_amp observeWithName:@"Signup" properties:@{@"lang": @"en", @"timeOfDay": @"evening"}];
+[_amp observeWithName:@"CheckoutAmount" properties:@{@"amount": amount}];
 ```
+Most likely, you will want to include `observe` events to capture metrics (that measures how well your application is to your business goals) or the contexts that are relevant to them. You would place this within action or delegate methods to capture events or at the startup of your app to capture contexts which will help to improve upon your business goals.
```
Comment (steve-si) [inline README.md]: ...events to capture `user context prior to decision-making and` capture metrics

File: README.md [hunk 2]
```diff
@@ -104,6 +117,7 @@ let colorDecision = amp.decide("ButtonColor", ["color":["blue", "orange", "green
 ``` ObjectiveC
 NSDictionary* decision = [_amp decideWithName:@"ButtonStyle" candidates:@{@"color": @[@"orange", @"blue", @"green"]} ttl:nil];
 ```
+Learning to make decisions to improve your metric is the key value Amp provides. Simply list the candidates, e.g. the style of the button, that are likely to make a difference for your metric, and Amp will help you learn from your data and make the best decision!
```
Comment (steve-si) [inline README.md]: Change `list` to `define`.

File: README.md [hunk 3]
```diff
@@ -134,10 +148,20 @@ amp.loadRules(timeout: timeoutInMilliseconds) { error in
...
+There are many ways in which you will want to use Amp. You may want to track how often each page is typically visited in your application or how far down a scrollable page your user scrolls.  If your application requires sign up and registration, you may want to track the number of taps on the sign up button because you may want to increase your user's sign up rate using Amp.  The possibilities are endless, and with Amp, not only will it track whatever you ask of it, it will also check the context in which these events occurred.  With this information, Amp will make the best decisions to improve on whatever business goals you have.
```
Comment (steve-si) [inline README.md]: Change `track the number of taps` to `track the occurrence of taps`.

**Review by steve-si — APPROVED** [review]
Left comments.

**Comment by olha-si** [comment]
Great comments! Updated docs.

**Pull request merged**